docs/scripts_md/database_files_update.md

```diff
@@ -0,0 +1,72 @@
+# NAME
+
+database\_files\_update.pl -- Updates path stored in `files` and
+`parameter_file` tables so that they are relative to data\_dir
+
+# SYNOPSIS
+
+perl database\_files\_update.pl `[options]`
+
+Available option is:
+
+\-profile: name of the config file in ../dicom-archive/.loris\_mri
+
+# DESCRIPTION
+
+This script updates the path stored in the `files` and `parameter_file`
+tables to remove the <\\$data\_dir> part of the path for security improvements.
+
+## Methods
+
+### get\_minc\_files($data\_dir, $dbh)
+
+Gets the list of MINC files to update the location in the `files` table.
+
+INPUT: data directory from the `Config` tables, database handle
```

Comment:
INPUTS (same comment applies to all INPUTS below)

Reply:
done